### DISCUSSION TITLE
How to customize the highlight of alpha's header?

### DISCUSSION THREAD

**Discussion by JoeMob** [ayamir/nvimdots#1193] [Q&A] [answered]
I changed highlight_overrides in user/configs/catppuccin.lua, but it doesn't work.
```lua
return {
	highlight_overrides = {
		latte = function()
			return {
				Normal = { fg = "#000000" },
				CmpBorder = { fg = "#000000" },
				AlphaHeader = { fg = "#FF0000" },
			}
		end,
	},
}
``` 
Is it because the highlight of AlphaHeader is set in utils/init.lua, and it's load after catppuccin?
If that's the problem, then how can I customize the highlight of AlphaHeader?

**Comment by charliie-dev** [comment, marked as answer]
ys, i think that's the case.

right now we dont have a easy solution for it, maybe consider make a fork and change it in `lua/modules/utils/init.lua`?
